api/adobe_vendor_id.py

```diff
@@ -892,9 +897,15 @@ def decode(self, authdata):
     def _decode(self, authdata):
         # First, decode the authdata without checking the signature.
         decoded = jwt.decode(
-            authdata, algorithm=self.ALGORITHM, options=dict(verify_signature=False)
+            authdata,
+            algorithms=[self.ALGORITHM],
+            options=dict(verify_signature=False, verify_exp=True),
         )
 
+        # Fail future JWTs as per requirements, pyJWT stopped doing this, so doing it manually
+        if "iat" in decoded and decoded["iat"] > time.time():
```

Comment:
I think we need to make sure that `time.time()` is giving us a unix timestamp. It does on most systems, but its not guaranteed. It looks like the [pyjwt library uses](https://github.com/jpadilla/pyjwt/blob/master/jwt/api_jwt.py#L133)
```python
now = timegm(datetime.now(tz=timezone.utc).utctimetuple())
``` 
to get the now value it uses in comparisons.